services/121-service/src/fsp-integrations/transaction-jobs/services/transaction-jobs-intersolve-visa.service.ts

```diff
@@ -173,6 +188,9 @@ export class TransactionJobsIntersolveVisaService {
       fundingTokenCode: intersolveVisaConfig.find(
         (c) => c.name === FspConfigurationProperties.fundingTokenCode,
       )?.value as string, // This must be a string. If it is not, the intersolve API will return an error (maybe).
+      cardDistributionByMail: intersolveVisaConfig.find(
+        (c) => c.name === FspConfigurationProperties.cardDistributionByMail,
+      )?.value as string, // This must be a string. If it is not, the intersolve API will return an error (maybe).
```

Comment:
```suggestion
      )?.value as string, 
```

That comment is not relevant, for this config property